packages/functions/workflows/validate-medical-necessity.ts

```
@@ -0,0 +1,593 @@
+  let confidence = 0;
+  const evidence: string[] = [];
+  let reasoning = '';
+
+  switch (criterion) {
+    case 'Clinical signs and symptoms support need for imaging':
+    case 'Patient condition poses significant health risk without intervention':
+      // Look for symptom and condition entities
+      const symptoms = medicalEntities.filter(e => 
+        e.Category === 'MEDICAL_CONDITION' && 
+        (e.Type === 'DX_NAME' || e.Traits?.some((t: any) => t.Name === 'SYMPTOM'))
+      );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)